src/Arcus.Scripting.ApiManagement/Scripts/Import-AzApiManagementOperationPolicy.ps1

```diff
@@ -0,0 +1,12 @@
+param(
+    [string][parameter(Mandatory = $true)] $ResourceGroup,
+    [string][parameter(Mandatory = $true)] $ServiceName,
+    [string][parameter(Mandatory = $true)] $ApiId,
+    [string][parameter(Mandatory = $true)] $OperationId,
+    [string][parameter(Mandatory = $true)] $PolicyFilePath
+)
+
+$apimContext = New-AzApiManagementContext -ResourceGroupName $ResourceGroup -ServiceName $ServiceName
```

Comment:
Let's include some error-handling in case the ApiManagement-instance cannot be found:
`Write-Error 'Specified API Management instance could not be found'`

Reply:
Hmm, I don't find any info/docs about any exception being thrown here, and when I try it, it doesn't throw either. 🤔

Reply:
I think you create a new instance of a type with this one but not validate ifit correspondence with n existing service.

Reply:
I used the output Boolean of the next command to write this error.